templates/bot/ts/workflow/src/cardActions/helloworldActionHandler.ts

```diff
@@ -0,0 +1,38 @@
+import { AdaptiveCards } from "@microsoft/adaptivecards-tools";
```

Comment:
Can we have consistent naming on action handler and its corresponding response card? like `helloWorldActionHandler.ts` and `helloWorldActionResponse.json`